bridge/opentracing/README.md

```diff
@@ -58,32 +58,40 @@ if s, ok := sc.(samplable); ok && s.IsSampled() {
 }
 ```
 
-### `SpanContext.TraceID`
-
-Return the underlying OpenTelemetry `Span.TraceID` value by converting a `bridgeSpanContext`.
+Any `SpanContext` method can be accessed as following:
```

Comment:
The  "### `SpanContext.IsSampled`" section is now redudant and can be removed. You can just consider adding `IsSampled() bool` to `spanContextProvider` below.

Reply:
Done.